database/youtube/css.json

```diff
@@ -1,34 +1,42 @@
 [
-    {
-        "name": "Kevin Powell",
-        "description": "It will give you idea about css. How to use css and many more things.",
-        "url": "https://www.youtube.com/@KevinPowell",
-        "category": "youtube",
-        "subcategory": "css",
-        "language": "english"
-    },
-    {
-        "name": "Online Tutorials",
-        "description": "It is great youtube channel for learning css in depth. By this youtube content you can make yourself expert in css.",
-        "url": "https://www.youtube.com/@OnlineTutorialsYT/featured",
-        "category": "youtube",
-        "subcategory": "css",
-        "language": "english"
-    },
-    {
-        "name": "DesignCourse",
-        "description": "This channel is great for learning css and frontend.",
-        "url": "https://www.youtube.com/@DesignCourse/featured",
-        "category": "youtube",
-        "subcategory": "css",
-        "language": "english"
-    },
-    {
-        "name": "Animation Coding",
-        "description": "A wide range of educational videos covering various topics such as animation, Motion Graphic Design, 3D animation, responsive website design, HTML, CSS, JavaScript, UX/UI design, After Effects, Photoshop, Adobe Illustrator, and much more.",
-        "url": "https://www.youtube.com/@AnimationCoding",
-        "category": "youtube",
-        "subcategory": "css",
-        "language": "english"
-    }
+  {
+    "name": "Kevin Powell",
+    "description": "It will give you idea about css. How to use css and many more things.",
+    "url": "https://www.youtube.com/@KevinPowell",
+    "category": "youtube",
+    "subcategory": "css",
+    "language": "english"
+  },
+  {
+    "name": "Online Tutorials",
+    "description": "It is great youtube channel for learning css in depth. By this youtube content you can make yourself expert in css.",
+    "url": "https://www.youtube.com/@OnlineTutorialsYT/featured",
+    "category": "youtube",
+    "subcategory": "css",
+    "language": "english"
+  },
+  {
+    "name": "DesignCourse",
+    "description": "This channel is great for learning css and frontend.",
+    "url": "https://www.youtube.com/@DesignCourse/featured",
+    "category": "youtube",
+    "subcategory": "css",
+    "language": "english"
+  },
+  {
+    "name": "Animation Coding",
+    "description": "A wide range of educational videos covering various topics such as animation, Motion Graphic Design, 3D animation, responsive website design, HTML, CSS, JavaScript, UX/UI design, After Effects, Photoshop, Adobe Illustrator, and much more.",
+    "url": "https://www.youtube.com/@AnimationCoding",
+    "category": "youtube",
+    "subcategory": "css",
+    "language": "english"
+  },
+  {
+    "name": "CSS Diner",
+    "description": "It's a fun game to learn and practice CSS selectors.",
+    "url": "https://flukeout.github.io/",
+    "category": "youtube",
```

Comment:
This resource doesn't belong to YOUTUBE could you please update the category accordingly?

Reply:
In which category I put it ?
my github username parma1111

Reply:
> This resource doesn't belong to YOUTUBE could you please update the category accordingly?

I think this calls for a new category "games”. What do you think @rupali-codes?

Reply:
@CBID2 not a game though, but @parma1111 can you put this in `UI Generators` in `Frontend` category

Reply:
> @CBID2 not a game though, but @parma1111 can you put this in `UI Generators` in `Frontend` category

> Ok I will put it there @rupali-codes

Reply:
> done @rupali-codes